capsules/classifier_safety_gear_openvino/README.md

```diff
@@ -0,0 +1,9 @@
+# About the Capsule
+## Usage
+This capsule is for detecting if a person wears safety gears (safety vest and safety hat).
+
+## Model
+###  Model File Origin
+The pretrained model was downloaded from the [Intel IoT Developer Program](https://software.intel.com/content/www/us/en/develop/topics/iot/reference-implementations/safety-gear-detector.html).
```

Comment:
@apockill The model I download from their repo is a caffe model. I convert it to OpenVino format. Should I mention this here?

Reply:
Yes definitely. 

For example, in this capsule: https://github.com/aotuai/capsule_zoo/tree/vcap-v0.2/capsules/classifier_mask_closeup

I show the exact command I used to do the optimization. It's not necessary, but it's helpful for people who are learning how to create their own capsules.